projects/keycloak/CredentialValidatorFuzzer.java

```diff
@@ -70,6 +72,11 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) {
           mockObject.getRealmModel(), mockObject.getUserModel(), mockObject.getCredentialInput());
     } catch (IllegalArgumentException e) {
       // Known exception
+    } catch (RuntimeException e) {
+      // Catching expected RuntimeException from json parsing problem
```

Comment:
Can you elaborate this please? What is the problem, and why does the fuzzer expect this?

Reply:
Ok, please add that as an inline comment in the fuzzer.